src/version.h

```diff
@@ -1,4 +0,0 @@
-#ifndef VERSION_H
```

Comment:
Why this file is removed? it's needed for development

Reply:
My bad, commited this by mistake.